components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -469,8 +469,8 @@ export class TypeORMWorkspaceDBImpl extends TransactionalDBImpl<WorkspaceDB> imp
             .andWhere("wsi.creationTime >= :periodStart", { periodStart: periodStart.toISOString() })
             .andWhere("wsi.creationTime <= :periodEnd", { periodEnd: periodEnd.toISOString() })
             .orderBy("wsi.creationTime", "DESC")
-            .offset(offset)
-            .limit(limit)
+            .skip(offset)
+            .take(limit)
```

Comment:
This caused some inconsistencies with the way we design our pagination in other API calls, so I aligned it with those.